src/page-response-impact/ValidDataWrapper.tsx

```diff
@@ -0,0 +1,90 @@
+import { sum } from "d3";
+import { pick } from "lodash";
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+
+import Colors from "../design-system/Colors";
+import { Facilities, Facility } from "../page-multi-facility/types";
+const POPULATION_DATA_ERROR_MSG =
+  "Impact could not be generated because one or more of your facilities does not have a staff or incarcerated population count. Please add population numbers and try again.";
+const LOCALE_DATA_ERROR_MSG =
+  "Impact could not be generated because locale information does not match across facilities";
```

Comment:
Nit: a period at the end of this one for consistency.

Reply:
Did we get this copy from @sychang? If not, I'm wondering if users will immediately grasp the meaning of "locale information" or if we should be more explicit?

Reply:
Yes, these came from the [PRD document](https://docs.google.com/document/d/1_jbpbsObUWkakyitmuIMaNKhq1YVU_fpo4j06ip8ets/edit?ts=5e95eb62#) and @sychang. But I agree with you though that we might need to be more explicit about what needs to be updated on the facilities. @sychang Any thoughts?